verbose_debugging_reports.md

```diff
@@ -134,12 +131,15 @@ otherwise the dictionary may include the following fields:
 * `source_site`: The site on which source was registered, e.g. `"https://source.example"`.
 * `trigger_debug_key`: The debug key in the trigger registration, omitted if not set.
 
+If `type` is [`source-success`](#source-success) or [`source-noised`](#source-nosied), the `body`
+dictionary may include `source_destination_limit` field if [destination limit](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#limiting-the-number-of-unique-destinations-covered-by-unexpired-sources)
+was exceeded.
+
 This table defines the fields in the `body` dictionary.
 
 | `type` | `attribution_destination`| `limit` | `source_debug_key` | `source_event_id` | `source_site` | `trigger_debug_key` |
```

Comment:
Shouldn't there be a column now for `source_destination_limit`?

Reply:
This field is only applicable to two fields, and is already noted separately.

Reply:
IMO it's harder to use the table if it's incomplete.

Reply:
Updated, thanks.